inference-engine/tests/functional/inference_engine/transformations/gelu_fusion.cpp

```diff
@@ -142,3 +142,83 @@ TEST(TransformationTests, GeluFusionPatternThree) {
+    std::shared_ptr<Function> f(nullptr), f_ref(nullptr);
+    {
+        auto data =
+            std::make_shared<opset7::Parameter>(element::f32, Shape{2, 2});
+
+        auto div_const =
+            opset7::Constant::create(element::f32, Shape{1}, {1.4149});
+        auto add_const =
+            opset7::Constant::create(element::f32, Shape{1}, {1.0});
+        auto mul_const =
+            opset7::Constant::create(element::f32, Shape{1}, {0.5});
+
+        auto div = std::make_shared<opset7::Divide>(data, div_const);
+        auto erf = std::make_shared<opset7::Erf>(div);
+        auto add = std::make_shared<opset7::Add>(erf, add_const);
+        auto mul_first = std::make_shared<opset7::Multiply>(data, add);
+        auto mul = std::make_shared<opset7::Multiply>(mul_first, mul_const);
+
+        f = std::make_shared<Function>(NodeVector{mul}, ParameterVector{data});
+
+        pass::Manager m;
+        m.register_pass<pass::InitNodeInfo>();
+        m.register_pass<pass::GeluFusionWithErfTwo>();
+        m.run_passes(f);
+        ASSERT_NO_THROW(check_rt_info(f));
+    }
+
+    {
+        auto data =
+            std::make_shared<opset1::Parameter>(element::f32, Shape{2, 2});
+        auto gelu = std::make_shared<opset7::Gelu>(data);
+        f_ref =
+            std::make_shared<Function>(NodeVector{gelu}, ParameterVector{data});
+    }
+
+    auto res = compare_functions(f, f_ref);
+    ASSERT_FALSE(res.first) << res.second;
+}
+
+TEST(TransformationTests, GeluFusionPatternTooShortDivConstValue) {
+    std::shared_ptr<Function> f(nullptr), f_ref(nullptr);
+    {
+        auto data =
+            std::make_shared<opset7::Parameter>(element::f32, Shape{2, 2});
+
+        auto div_const =
+            opset7::Constant::create(element::f32, Shape{1}, {1.4142});
+        auto add_const =
+            opset7::Constant::create(element::f32, Shape{1}, {1.0});
+        auto mul_const =
+            opset7::Constant::create(element::f32, Shape{1}, {0.5});
+
+        auto div = std::make_shared<opset7::Divide>(data, div_const);
+        auto erf = std::make_shared<opset7::Erf>(div);
+        auto add = std::make_shared<opset7::Add>(erf, add_const);
+        auto mul_first = std::make_shared<opset7::Multiply>(data, add);
+        auto mul = std::make_shared<opset7::Multiply>(mul_first, mul_const);
+
+        f = std::make_shared<Function>(NodeVector{mul}, ParameterVector{data});
+
+        pass::Manager m;
+        m.register_pass<pass::InitNodeInfo>();
+        m.register_pass<pass::GeluFusionWithErfTwo>();
+        m.run_passes(f);
+        ASSERT_NO_THROW(check_rt_info(f));
+    }
+
+    {
+        auto data =
+            std::make_shared<opset1::Parameter>(element::f32, Shape{2, 2});
+        auto gelu = std::make_shared<opset7::Gelu>(data);
+        f_ref =
+            std::make_shared<Function>(NodeVector{gelu}, ParameterVector{data});
+    }
+
+    auto res = compare_functions(f, f_ref);
+    ASSERT_FALSE(res.first) << res.second;
```

Comment:
I think it's better to have reference function to be a copy of the original and put ASSERT_TRUE here.